lib/reline/line_editor.rb

```diff
@@ -740,9 +740,20 @@ def add_dialog_proc(name, p, context = nil)
     move_cursor_down(dialog.vertical_offset)
     Reline::IOGate.move_cursor_column(dialog.column)
     dialog.contents.each_with_index do |item, i|
+      fg_color = 37
       if i == pointer
-        bg_color = '45'
+        if dialog_render_info.pointer_fg_color
```

Comment:
How about setting the default values when initializing the reline like other configs?

https://github.com/ruby/reline/blob/5b3fc02196061e3026dbf6a2553b35142a2fa5dc/lib/reline.rb#L546-L551

Then we don't need these if statements because the `dialog_render_info` should always have a value from the config default. It will also help users understand the default by calling `Reline.dialog_default_fg_color` directly.

If we really worry about the values being `nil` by accident, we should validate them at the setter.